python/lsst/ip/isr/isrTask.py

```diff
@@ -189,71 +188,123 @@ class IsrTaskConfig(pexConfig.Config):
-             "also have to interpolate them before flat-fielding."),
+        doc="Fit the overscan in a piecewise-fashion to correct for bias jumps?",
         default=False,
     )
-    fluxMag0T1 = pexConfig.Field(
-        dtype=float,
-        doc="The approximate flux of a zero-magnitude object in a one-second exposure",
-        default=1e10,
+    overscanBiasJumpKeyword = pexConfig.Field(
+        dtype=str,
+        doc="Header keyword containing information about devices.",
+        default="NO_SUCH_KEY",
     )
-    keysToRemoveFromAssembledCcd = pexConfig.ListField(
+    overscanBiasJumpDevices = pexConfig.ListField(
         dtype=str,
-        doc="fields to remove from the metadata of the assembled ccd.",
-        default=[],
+        doc="List of devices that need piecewise overscan correction.",
+        default=(),
     )
+    overscanBiasJumpLocation = pexConfig.Field(
+        dtype=int,
+        doc="Location of bias jump along y-axis.",
+        default=0,
+    )
+
+    doAssembleCcd = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc="Assemble amp-level exposures into a ccd-level exposure?"
+    )
+    assembleCcd = pexConfig.ConfigurableField(
+        target=AssembleCcdTask,
+        doc="CCD assembly task",
+    )
+
     doAssembleIsrExposures = pexConfig.Field(
         dtype=bool,
         default=False,
         doc="Assemble amp-level calibration exposures into ccd-level exposure?"
     )
-    doAssembleCcd = pexConfig.Field(
+    doTrimToMatchCalib = pexConfig.Field(
         dtype=bool,
-        default=True,
-        doc="Assemble amp-level exposures into a ccd-level exposure?"
+        default=False,
+        doc="Trim raw data to match calibration bounding boxes?"
     )
-    expectWcs = pexConfig.Field(
+
+    doBias = pexConfig.Field(
         dtype=bool,
+        doc="Apply bias frame correction?",
         default=True,
-        doc="Expect input science images to have a WCS (set False for e.g. spectrographs)"
     )
+    biasDataProductName = pexConfig.Field(
+        dtype=str,
+        doc="Name of the bias data product",
+        default="bias",
+    )
+
+    doVariance = pexConfig.Field(
+        dtype=bool,
+        doc="Calculate variance?",
+        default=True
+    )
+    gain = pexConfig.Field(
+        dtype=float,
+        doc="The gain to use if no Detector is present in the Exposure (ignored if NaN)",
+        default=float("NaN"),
+    )
+    readNoise = pexConfig.Field(
+        dtype=float,
+        doc="The read noise to use if no Detector is present in the Exposure",
+        default=0.0,
+    )
+    doEmpiricalReadNoise = pexConfig.Field(
+        dtype=bool,
+        default=False,
+        doc="Calculate empirical read noise instead of value from AmpInfo data?"
+    )
+
     doLinearize = pexConfig.Field(
         dtype=bool,
         doc="Correct for nonlinearity of the detector's response?",
         default=True,
     )
+
```

Comment:
Same comment as above.  Looking at this, it seems some configs are being logically grouped, but if that's the case, it would be kind of nice to have a comment saying what the groups are.  This is optional, I guess.

Reply:
I've added comments to explain the groups.  I needed some way to organize the configuration options for myself, given the large number of entries.  The previous ordering was not clear, and this will hopefully minimize future searching.